app/sidebar.py

```diff
@@ -28,6 +29,14 @@
         html.H6(
             id="data_description", style={"textAlign": "left", "color": colors["text"]}
```

Comment:
In my opinion, there is a simpler way of doing this without adding new keyword, ``citation`` to the JSON file or adding more code to the file. Use the pre-existing description key and change its type from `html.H6` to `dcc.Markdown` component. You can then use Markdown in the JSON file to add the link, for example,

     "name": "Ca₃SiO₄Cl₂",
     "description": "A calcium-silicate commonly used in LEDs. [Hansen et al. DOI: 10.1021/ic020647f](https://pubs.acs.org/doi/abs/10.1021/ic020647f)",